pkg/util/cluster.go

```diff
@@ -18,13 +21,17 @@ type ClusterInfo interface {
 	IsRunningLocally() bool
 	GetDomain() string
 	IsManagedByOLM() bool
+	IsControlPlaneHighlyAvailable() bool
```

Comment:
This is not in use in the code. Maybe we want to remove it to simplify the interface?

Reply:
we are not using it now with kubevirt, but I'm pretty sure we will start using it in the future with other components like CNAO which is scheduling also on masters.